examples/layer-browser/package.json

```diff
@@ -5,7 +5,7 @@
     "build": "webpack app bundle.js --env.local --display-error-details"
   },
   "dependencies": {
-    "deck.gl": "^4.0.0-beta.3",
+    "deck.gl": "^4.0.0-rc1",
```

Comment:
Yes. I noticed that yesterday. @Pessimistress 's ocs-lite got updated to this version automatically. 

I'm going to republish it today after @gnavvy 's change got landed